docker/secretflow-gpu.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM nvidia/cuda:11.8.0-cudnn8-devel-ubuntu20.04
+
+RUN  apt-get update \
+     && apt-get install -y libcudnn8=8.6.0.163-1+cuda11.8 --allow-downgrades --allow-change-held-packages  \
+     && apt-get install -y python3.8 --allow-downgrades --allow-change-held-packages   
+     && apt-get install -y python3-pip --allow-downgrades --allow-change-held-packages 
+
+RUN if [ ! -e /usr/bin/python ]; then ln -sf /usr/bin/python3.8 /usr/bin/python; fi
+
+RUN if [ ! -e /usr/bin/python3 ]; then ln -sf /usr/bin/python3.8 /usr/bin/python3; fi
+
+#install the dependencies of cuda11
+#you are supposed to  add the mirror source of pypi to accelerate installation of nvidia packages of cuda11, 
+#if not, the building of images are prone to fail very much 
+RUN pip install nvidia-cublas-cu11 nvidia-cuda-cupti-cu11 nvidia-cuda-nvcc-cu11 \
+    nvidia-cuda-nvrtc-cu11 nvidia-cuda-runtime-cu11 install nvidia-cudnn-cu11 \
+    nvidia-cufft-cu11  nvidia-curand-cu11  nvidia-cusolver-cu11 \
+    nvidia-cusparse-cu11 nvidia-nccl-cu11  nvidia-nvtx-cu11 
+
+# install the gpu version of jax and jaxlib based cuda11
+# the site of https://storage.googleapis.com/jax-releases/jax_cuda_releases.html is very necessary
+# ref to https://github.com/google/jax#pip-installation-gpu-cuda-installed-via-pip-easier
+RUN pip install --upgrade "jax[cuda11_pip]"==0.4.1 -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html \
```

Comment:
Will jax&jaxlib be reinstalled with another version after `pip install -U secretflow`?

Reply:
No, base on the CUDA and cudnn, they have strict dependencies of version.
ref to:
tf:https://tensorflow.google.cn/install/source#gpu
torch:https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
jax:https://storage.googleapis.com/jax-releases/jax_cuda_releases.html

Reply:
maybe jax&&jaxlib == 0.4.7 could be ok?

Reply:
I will test the combination of latest jax and jaxlib soon.

Reply:
I try to use the latest version of jax and jaxlib, However it could't run the code of jax.
More  appropriate and updated version of jax and jaxlib need to be test so that we can get the better Dockerfile.